plugins/depthCamera/src/DepthCamera.cc

```diff
@@ -12,14 +12,18 @@
 #include <GazeboYarpPlugins/ConfHelpers.hh>
 #include <yarp/os/Log.h>
 #include <yarp/os/LogStream.h>
+#include <yarp/os/LogComponent.h>
+
 #include <gazebo/sensors/DepthCameraSensor.hh>
 #include <yarp/dev/PolyDriver.h>
 #include <yarp/dev/IMultipleWrapper.h>
 
 
 
 GZ_REGISTER_SENSOR_PLUGIN(gazebo::GazeboYarpDepthCamera)
-
+namespace {
+    YARP_LOG_COMPONENT(GAZEBODEPTH, "gazebo-yarp-plugins.plugins.GazeboYarpDepthCamera DepthCamera.cc")
```

Comment:
This seems to follow a different style, is this intended?

Reply:
no it was a refuse, i'll check them all :)